server/models/video/video-abuse.ts

```diff
@@ -382,6 +411,7 @@ export class VideoAbuseModel extends Model<VideoAbuseModel> {
     return {
       id: this.id,
       reason: this.reason,
+      predefinedReasons: this.predefinedReasons,
```

Comment:
I think you should also use strings here for consistency

Reply:
it is harder though.